example/lib/src/ui/ble_status_screen.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
+
+class BleStatusScreen extends StatelessWidget {
+  const BleStatusScreen({@required this.status, Key key})
+      : assert(status != null),
+        super(key: key);
+
+  final BleStatus status;
+
+  String determineText(BleStatus status) {
+    switch (status) {
+      case BleStatus.unsupported:
+        return "This device does not support BLE";
+      case BleStatus.unauthorized:
+        return "Authorize the FlutterReactiveBle example app to use bluetooth and location";
+      case BleStatus.poweredOff:
+        return "Ble is powered off on your device turn it on";
```

Comment:
Spell abbreviations in all capital ("BLE") or use "Bluetooth" (first letter capitalized), please.

And whatever you choose ("Bluetooth" or "BLE"), consider using it **consistently** in all these messages, please.